apogee/tools/read.py

```diff
@@ -584,6 +590,15 @@ def rcsample(main=False,dr=None,xmatch=None,
         data= data[m1]
         astroNNdata= astroNNdata[m2]
         data= _add_astroNN_ages(data,astroNNdata)
+    if use_astroNN or kwargs.get('astroNN', False) or use_astroNN_orbits:
+        _warn_astroNN_orbits()
+        astroNNdata= astroNN()
+        # Match on (ra,dec)
+        m1,m2,_= _xmatch(data,astroNNdata,maxdist=2.,
+            colRA1='RA',colDec1='DEC',colRA2='ra_apogee',colDec2='dec_apogee')
+        data= data[m1]
+        astroNNdata= astroNNdata[m2]
+        data= _add_astroNN_orbts(data,astroNNdata)
```

Comment:
typo I think? Please test to make sure this works!

Reply:
this was indeed a typo, but there was also some issues with this which are now fixed